documentation/customisation.md

```diff
@@ -0,0 +1,139 @@
+# Dockerfile Customisation
+
+A custom Docker file might be required in certain circumstances. These include (but aren't limited to)
+
+- Connecting to a properietary database (DB2, Oracle, etc.)
+- Adding a hotfix as instructed by our support team
+- Adding a custom plugin
+- etc.
+
+The process to customise a docker image for your needs is described below
+
+## XL Deploy
+
+When creating a Dockerfile with custom resources added, always remember that the owning user:group combination _must_ be `10001:0`, or else XL Deploy will not be able to read the files. 
+
+To start off with, create a `Dockerfile` that resembles the following. 
+
+Please note that certain JARS will go into certain paths. You shouldn't add a Oracle JAR to the `ext/` folder, for example. If you are unsure where a JAR should be added, please get in touch with our support team. 
+
+NOTE: `${APP_HOME}` points to the path `/opt/xebialabs/xl-deploy-server` by default
+
+```docker
+FROM xebialabs/xl-deploy:9.5.0
```

Comment:
- prefer to use :9.5 or :9.0 tags which point to latest maintenance version .. and now i did testing using :9.0 as :9.5 not yet released.

Reply:
this is just an example. these docs are supposed to be released next release when 9.5.0 will be available